quartz/components/Explorer.tsx

```diff
@@ -116,7 +117,7 @@ export default ((userOpts?: Partial<Options>) => {
             <polyline points="6 9 12 15 18 9"></polyline>
           </svg>
         </button>
-        <div class="explorer-content" aria-expanded={false}>
+        <div id="explorer-content">
```

Comment:
A while back all components were changed to use classes over id's to allow non-atomic usage.

Reply:
I see what you mean, but the issue with this is that aria-attributes actually requires ID's and don't work with classes.

Reply:
One way to deal with this would be to generate an idea for these fields. The way Overflowlist does perhaps.

Reply:
Except they don't. They require a unique identifier field per tag. This does **not** have to be the `id` field.

Reply:
I do think that `aria-controls` require the `id` tag of the element. Here are the specifications

[aria-controls specification](https://www.w3.org/TR/wai-aria-1.3/#aria-controls)
[Id reference list value type](https://www.w3.org/TR/wai-aria-1.3/#valuetype_idref_list)

This element in particular requires it because the button has `aria-controls="explorer-content"`.

Reply:
Alright, but that still doesn't address the issue I raised.

Your proposed changes remove the classes and add id's. Removing the classes will break component functionality.

In addition, id's must be unique for aria to function properly, per your own sources. Currently, components are non-atomic. If you want to add id's for the purpose of accessibility, please ensure uniqueness in case of multiple components of the same type. (Ideally consistent as well, e.g. no random generated appended text.)

Reply:
Yeah, Like I said I could generate them if you want to support multiple of the same component.
It would also require the inline script to change a bit so it targets the correct one, but I think we can manage that.